src/JsonDocument/Testing/AssertJsonDocumentTrait.php

```diff
@@ -11,6 +11,21 @@
      * @param JsonDocument $expected
      * @param JsonDocument $actual
      */
+    private function assertJsonDocumentOrderlessEquals(
```

Comment:
Naam is nogal vaag. Misschien eerder iets als `assertJsonDocumentHasSamePropertiesInAnyOrder`? Ook niet perfect natuurlijk... Ik sta ook open voor andere suggesties :)